Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -102,6 +102,22 @@ public extension URL {
         }
     }
 
+    /// SwifterSwift: Generates new URL that does not have scheme.
+    ///
+    ///        let url = URL(string: "https://domain.com")!
+    ///        print(url.droppedScheme()) // prints "domain.com"
+    public func droppedScheme() -> URL? {
+        // https://github.com/SwifterSwift/SwifterSwift/issues/464
+        if let scheme = self.scheme {
+            return URL(string: String(self.absoluteString.dropFirst(scheme.count + 3)))
+        }
+
+        guard host != nil else {
+            return self
+        }
+
+        return URL(string: String(absoluteString.dropFirst(2)))!
```

Comment:
If the return is optional, why force unwrap here?

Reply:
It would be nice to extract the inner expression

Reply:
@SD10 I've extracted the inner expression, in both cases where `URL(string:)` is used.

@LucianoPAlmeida removed force unwrap.